src/DetectionResult.cs

```diff
@@ -41,7 +41,7 @@ public DetectionResult(DetectionDetail detectionDetail)
         /// <summary>
         /// All results
         /// </summary>
-        public IList<DetectionDetail> Details { set; get; }
+        public IList<DetectionDetail> Details { get; private set; }
```

Comment:
Good idea, but this is a breaking change. So please leave this now as it is.

Reply:
Ok, np :)